kratos/python/add_containers_to_python.cpp

```diff
@@ -242,6 +242,10 @@ void  AddContainersToPython(pybind11::module& m)
     .def("__str__", PrintObject<Variable<Quaternion<double> >>)
     ;
 
+    py::class_<Variable<Parameters::Pointer>, VariableData>(m, "ParametersVariable")
```

Comment:
@RiccardoRossi shouldn't this just be `Parameters` and not `Parameters::Pointer`?

Reply:
From my point of view, it can be `Parameters` or `Parameters::Pointer` I will wait your opinion to apply changes

Reply:
I mean that `Parameters` already is sort of a shared pointer to the underlying json, therefore it is sufficient to use it by-value (according to @RiccardoRossi )